main/src/ca/uwaterloo/flix/api/lsp/Request.scala

```diff
@@ -54,6 +54,16 @@ object Request {
     */
   case class RemPkg(requestId: String, uri: String) extends Request
 
+  /**
+    * A request to add (or update) the JAR at the given uri with the given binary data.
+    */
+  case class AddJar(requestId: String, uri: String) extends Request
```

Comment:
I assume if the jar is *updated* the class loader will not reload it. Probably a minor problem, but the comment is wrong.